Sources/AccountSDKIOSWeb/Simplified Login UI/SimplifiedLoginNebula.swift

```diff
@@ -0,0 +1,397 @@
+import SwiftUI
+import UIKit
+
+public class SimplifiedLoginManager {
+    public func presentSimplifiedLogin(presentingViewController: UIViewController) {
```

Comment:
Thinking about having a function like this as the main access point of Simplfied Login.

SimplifiedLoginManager is probably not a good name. Maybe SimplifiedLoginUIFactory.
SDK consumers will need to inject a User instance on constructing this class aswell.

Reply:
I wonder if this brings additional benefits to developers. What about delivering just VC and let them create it and add to their navigation flow?

Reply:
Also I encourage to have SimplifiedLoginViewController with public methods in one file, and all other  accompanying views in different files. Just to make it simple and clear for devs